src/_locales/es/messages.json

```diff
@@ -2,5 +2,214 @@
     "extensionName": {
         "message": "LibRedirect",
         "description": "name of the extension"
+    },
+    "switchInstance": {
+        "message": "Cambiar instancia",
+        "description": "used in manifest.json as shortcut description"
+    },
+    "settings": {
+        "message": "Ajustes",
+        "description": "used in the popup"
+    },
+    "general": {
+        "message": "General",
+        "description": "used in the settings page"
+    },
+    "translate": {
+        "message": "Traductor",
+        "description": "used in the settings page"
+    },
+    "search": {
+        "message": "Búsqueda",
+        "description": "used in the settings page"
+    },
+    "ytmusic": {
+        "message": "YT Music",
+        "description": "used in the settings page"
+    },
+    "wikipedia": {
+        "message": "Wikipedia",
+        "description": "used in the settings page"
+    },
+    "medium": {
+        "message": "Medium",
+        "description": "used in the settings page"
+    },
+    "tor": {
+        "message": "Tor",
+        "description": "used in the settings page"
+    },
+    "theme": {
+        "message": "Tema",
+        "description": "used in the settings page"
+    },
+    "light": {
+        "message": "Claro",
```

Comment:
I am unsure of this alone. Everything seems fines.

Reply:
The Light string alone.

Reply:
I think it's correct: https://st.privacydev.net/?engine=google&text=Tema%20Claro&sl=es&tl=en

Reply:
Okay, Great.